### ISSUE TITLE
GameGuru MAX - Request - Workflow Issue

### ISSUE THREAD

**Issue by ArgentArts** [TheGameCreators/GameGuruRepo#1689]
With this last alpha of MAX, I did a clean install with no old projects to load. So, I faced it as if I were a brand new person facing MAX for the first time. I have to say, it did not go well. I found myself in what felt like an endless loop of attempting to get to the editor so I could start to build a level. Part of that was because of my previous experiences with MAX. But most of it was because of getting lost at one point due to MAX not being clear on how to proceed. I'll try to illustrate the issue.

When you first begin a fresh install of MAX, you see this:

![max1](https://user-images.githubusercontent.com/36735507/146815811-e8601045-7351-4376-8351-15d38326d074.png)

Now, I had already been through several tests trying various things, so ignore the three test games in the My Games area. Other than that, this is basically what a new user would see. After clicking on a New Game you are taken to the Storyboard area:

![max2](https://user-images.githubusercontent.com/36735507/146816006-b8203c45-08f2-4609-849a-cb7d5340827e.png)

So far, so good. Even the first level has a large Click Here! to attract the end-user, telling them what to do. This is a good start, imo. Which leads us to the next screen:

![max3](https://user-images.githubusercontent.com/36735507/146816117-9cb507fb-f372-4399-b1a4-755aaf74c7cc.png)

Here, we're given a pre-generated terrain and a set of controls to alter that or to choose another. I decided I wanted an Empty so I could build an in-door level:

![max4](https://user-images.githubusercontent.com/36735507/146816224-bb4cb6f0-83e9-494f-ba7d-26d8a1179767.png)

So far, so good ... but now what? I pressed the back arrow at the top-left. Nope. That sends up a dialog stating the terrain has not been saved and asks are you sure? Yes, No, Cancel. So, I think, "Maybe I now need to save this terrain first, even though it's an empty." So, I do that. But then I am right back to this same screen, not at the editor. Nothing's changed. So, I click the back arrow at the top-left again and get the exact same dialog asking me the exact same thing. Ugh! This time I press Yes, I am sure. Let's go! And this brings me back to the Storyboard page, my empty level "lost". Now I am frustrated. So, I click on the Click Here! again, go back to the generated terrain, and click Empty again and am left to wonder HOW DO I GET TO THE EDITOR!!!

So, maybe I'm not the brightest bulb in the pack, but I try different things and just ... get ... frustrated. I don't see a way to proceed. I don't see a way to get out of this screen and into the editor with an empty so I can start to create a level. Talking to a few people on the Discord channel solved this one - I had to click the Generate Terrain for Level button to move on ... sigh.

See, here's the thing - in my mind, an "empty" level was "empty" so did it need to be "generated"? Perhaps it did, but MAX didn't tell me that. So far, each step of the way, MAX told me to do something. It said, "Click Here!" and then, "Click Here!", but when I got to this screen, it didn't give me any indication at all what to do in order to move forward. Saving the terrain, which seemed logical, didn't do anything to move me forward. I didn't want to load a terrain. And, heck, I wanted an "Empty" so I wasn't even thinking in terms of "terrains" anyway. So, "generating" one wasn't even on my radar. 

The point is, this workflow is not intuitive in the least. It may work for those that are already familiar with it, but if it's a first-time user, they are going to go nuts, I think, and get frustrated and want to get their money back because they won't even be able to get to the editor to begin creating something, which is the point of this software, after all. And that would be a shame. If you're going to keep this workflow (and I really hope you guys don't), then at least create a big, freakin' button at this last screen that screams PRESS ME TO PROCEED! so the end-user knows this is the last step to get to the editor.

However, in all honesty, I think it's best to just open at the editor like any other piece of software does. I'd previous opened a request about that here:

https://github.com/TheGameCreators/GameGuruRepo/issues/1678

If MAX just started at the editor, then the end-user could go to the menu and select New Project. The New Project window could be exactly what this is now, if you like. Or you could have two modes - New Project - Guided (which would be this and for newbies) and New Project - Advanced (for the rest of us that don't need to go step-by-step having our hands held all the way through - for those of us that can summon the storyboard when we need it, etc.). 

Whatever you guys do, think of the workflow the end-user needs to go through to actually work on creating a game. In this case, try to see MAX through the eyes of someone who has never used the software before. I had used it and these initial screens almost drove me to uninstall it because I could not get to the editor to create a new project. :)

**Comment by Necrym59** [comment]
I agree its getting very convoluted in its workflow logic process. I use to work in Human User Interface Processing for a lot of business apps. Most would think the 'storyboard' should be the main project screen everything should stem from there eg:  (Menu - Project -New, Open, Delete etc...) On 'New' ..then generate a storyboard. On 'Open' use a thumbmail selection window or panel. 'Delete is self explanatory. 'Settings' for the Storyboard eg. (fps, rpg,other, information and game settings) etc all happens from within the Project/Stroryboad section. It all stems from there - (extra leves/videos/menus/ Importer* etc). The Importer is also used with the level edit section. But should be called in standalone mode for multiple importing without having to leap in and out of launching a level for every import. Also when a levels are saved it should ask to return you back to the storyboard for your next process/selection or to continue editing. 
Secondly the level create should open a tabbed panel for all the biom settings when a biom is selected only then showing the specific options for that biom to be tweaked as required and the other tabs for the existing lanscape tweaking. With a bigger highlighted GENERATE LEVEL button.
